core/experiments/wp-cli.php

```diff
@@ -31,18 +31,21 @@ public function activate( $args, $assoc_args ) {
 
 		// Activate experiment callback.
 		$activate = function ( $site = '', $id = null ) use ( $args, $is_network ) {
-			$success = 'Experiment activated successfully';
-			$error = 'Cannot activate experiment';
+			$experiments = explode( ',', $args[0] );
+			$plural = count( $experiments ) > 0 ? 's' : '';
+			$success = 'Experiment' . $plural . ' activated successfully';
+			$error = 'Cannot activate experiment' . $plural;
```

Comment:
In case `is_network === true` it will run for every website.
Consider preparing it outside the function.